src/agent/Cargo.toml

```diff
@@ -83,6 +83,8 @@ image-rs = { git = "https://github.com/confidential-containers/guest-components"
 tempfile = "3.1.0"
 test-utils = { path = "../libs/test-utils" }
 which = "4.3.0"
+rstest = "0.18.0"
```

Comment:
I guess we need the Cargo.lock updated again for this commit?

Reply:
Done.